packages/web-client/app/components/card-pay/deposit-workflow/confirmation/index.ts

```diff
@@ -35,9 +35,9 @@ class CardPayDepositWorkflowConfirmationComponent extends Component<WorkflowCard
 
   get receivedToken() {
     if (this.selectedTokenSymbol === 'CARD') {
-      return new TokenDisplayInfo('CARD.CPXD');
+      return new TokenDisplayInfo(this.layer2Network.bridgedCardTokenSymbol);
```

Comment:
This is also going to be awkwardly wrong on Sokol for now.

Reply:
It is 😓